webpack/assets/javascripts/react_app/common/globalIdHelpers.js

```diff
@@ -7,6 +7,8 @@
  * together into a single string and encoded as base64.
  */
 
+import { Buffer } from 'buffer';
```

Comment:
Looks like something we can already merge in a preparation PR, correct?

Reply:
https://github.com/theforeman/foreman/pull/9909